salt/modules/redismod.py

```diff
@@ -69,6 +69,17 @@ def _sconnect(host=None, port=None, password=None):
     return redis.StrictRedis(host, port, password=password)
 
 
+def _get_kwargs_opts(**options):
```

Comment:
Sure @cachedout, I will let you know when this is ready to be reviewed! I am again in transit and I will not have the time required over the next several days. Thanks for patience!